cmd/bucket-targets.go

```diff
@@ -125,8 +125,10 @@ func (sys *BucketTargetSys) SetTarget(ctx context.Context, bucket string, tgt *m
 
 	tgts := sys.targetsMap[bucket]
 	newtgts := make([]madmin.BucketTarget, len(tgts))
+	labels := make(map[string]string)
```

Comment:
we could use a `map[string]struct{}` we only need to know that a label has been used.

Reply:
done